Scotch include dir not listed when building unit tests

After successfully building `dolfinx` I am trying to compile the unit tests, but I get
```$ make unittests 
[ 94%] Built target dolfin
Scanning dependencies of target unittests
[ 95%] Building CXX object test/unit/CMakeFiles/unittests.dir/main.cpp.o
[ 96%] Building CXX object test/unit/CMakeFiles/unittests.dir/common/SubSystemsManager.cpp.o
In file included from /home/mleoni/opt/Xfenics/install/include/dolfin/graph/dolfin_graph.h:18:0,
                 from /home/mleoni/opt/Xfenics/install/include/dolfin.h:10,
                 from /home/mleoni/opt/Xfenics/source/dolfinx/cpp/test/unit/common/SubSystemsManager.cpp:23:
/home/mleoni/opt/Xfenics/install/include/dolfin/graph/SCOTCH.h:21:10: fatal error: ptscotch.h: No such file or directory
 #include <ptscotch.h>
          ^~~~~~~~~~~~
compilation terminated.
```
Toggling the verbose option I see that when I am compiling dolfin the compile command includes `-isystem /usr/include/scotch`, while the command used to build the unit tests does not.

--- a/cpp/dolfin/graph/SCOTCH.cpp
+++ b/cpp/dolfin/graph/SCOTCH.cpp
@@ -11,9 +11,8 @@
 #include <dolfin/common/MPI.h>
 #include <dolfin/common/Set.h>
 #include <dolfin/common/Timer.h>
-#include <dolfin/mesh/CellType.h>
-
 #include <dolfin/common/log.h>
+#include <dolfin/mesh/CellType.h>
 #include <map>
 #include <numeric>
 #include <set>

--- a/cpp/dolfin/graph/SCOTCH.h
+++ b/cpp/dolfin/graph/SCOTCH.h
@@ -7,15 +7,16 @@
 #pragma once
 
 #include "Graph.h"
-#include <cstddef>
 #include <cstdint>
 #include <dolfin/common/MPI.h>
-#include <dolfin/common/types.h>
 #include <map>
-#include <set>
 #include <string>
+#include <utility>
 #include <vector>
 
+// Developer note: ptscotch.h is not part of the public interface,.
+// therefore this header (SCOTCH.h) should not be placed in the public
+// interface of DOLFIN
 extern "C"
 {
 #include <ptscotch.h>

--- a/cpp/dolfin/graph/SCOTCH.h
+++ b/cpp/dolfin/graph/SCOTCH.h
@@ -14,7 +14,7 @@
 #include <utility>
 #include <vector>
 
-// Developer note: ptscotch.h is not part of the public interface,.
+// Developer note: ptscotch.h is not part of the public interface,
 // therefore this header (SCOTCH.h) should not be placed in the public
 // interface of DOLFIN
 extern "C"

--- a/cpp/dolfin/graph/dolfin_graph.h
+++ b/cpp/dolfin/graph/dolfin_graph.h
@@ -15,4 +15,3 @@ namespace dolfin
 #include <dolfin/graph/BoostGraphOrdering.h>
 #include <dolfin/graph/Graph.h>
 #include <dolfin/graph/GraphBuilder.h>
-#include <dolfin/graph/SCOTCH.h>

